src/etc/inc/util.inc

```diff
@@ -1547,15 +1547,20 @@ function mwexec_bg($command, $clearsigmask = false) {
 	return mwexec($command, false, $clearsigmask, true);
 }
 
+function unlink_silent($fn) {
+	@unlink($fn);
+}
+
 /* unlink a file, or pattern-match of a file, if it exists
    if the file/path contains glob() compatible wildcards, all matching files will be unlinked
-   if no matches, no error occurs */
+   if no matches, no error occurs
+   attempts to unlink directories will fail silently */
 function unlink_if_exists($fn) {
 	$to_do = glob($fn);
 	if (is_array($to_do) && count($to_do) > 0) {
-		array_map("unlink", $to_do);
```

Comment:
What do you think about add @array_map here instead of creating this new function?